extensions/wikia/ContentReview/controllers/ContentReviewApiController.class.php

```diff
@@ -62,15 +65,7 @@ public function submitPageForReview() {
 	public function enableTestMode() {
 		$this->isValidPostRequest( $this->request, $this->wg->User );
 
-		$pageId = $this->request->getInt( 'pageId' );
-
-		$title = Title::newFromID( $pageId );
-		if ( $title === null || !$title->isJsPage() ) {
-			throw new NotFoundApiException( "JS page with ID {$pageId} does not exist");
-		}
-
-		$submitUserId = $this->wg->User->getId();
-		if ( !$submitUserId > 0 || !$this->canUserSubmit( $title )	) {
+		if ( !$this->wg->User->isLoggedIn() ) {
 			throw new PermissionsException( 'edit' );
```

Comment:
This now only checking logged in, so the permission specified in the exception shouldn't be `edit`.